### PR TITLE
volume controller color parameters

### PR DESCRIPTION
**Pull request by dmitrykcom**

### CODE REVIEW

**Comment by lental** [comment]
@chibzu  for your input, I know you sent an email out a while ago

File: skin.json [hunk 1]
```diff
@@ -137,6 +137,10 @@
     "timeToShow": "10"
   },
   "controlBar": {
+    "volumeView": {
```
Comment (chibzu) [inline skin.json]: View is kind of a loaded term, can we go with volumeControl or volumeBar or something along those lines.

File: skin.json [hunk 2]
```diff
@@ -137,6 +137,10 @@
...
+      "progressColor": "#FFFFFFFF",
+      "thumbColor": "#FFFFFFFF"
```
Comment (chibzu) [inline skin.json]: Is "thumb" the mobile terminology for the draggable element?

**Comment by chibzu** [comment]
The skin schema needs to be updated as well.

File: skin.json (hunk 2 above)
Comment (dmitrykcom) [inline skin.json]: Believe it or not it IS on Android. But yeah, we can come up with something
better.

**Comment by lental** [comment]
@chibzu  how do you feel about this? Dmitry boiled down to a single color, with renames and schema

**Comment by lental** [comment]
okay, adding this in.  please comment or talk to us if you want to change it

**Pull request merged**